docs/stack/subgraph/queries.md

```diff
@@ -0,0 +1,346 @@
+You can use GraphQL queries to get quick info from [DAOstack Sunbgraph](https://thegraph.com/explorer/subgraph/daostack/v30_1) hosted on GraphExplorer.
+
+This guide explains how to use GraphQL queries to get single or multiple Entities and sort, filter and paginate them.
+
+The full list of `Subgraph Entities` cached by DAOstack subgraph can be found [here](../Entity)
+
+## General Guidelines
+
+  1. All queries must be wrapped inside `query {}` object.
+  2. Entity name is same as provided in [Entity list](../Entity) but in `lower case`.
```

Comment:
starts with lower case letter (but camel case in general so should be `lowerCase` or so)